src/electron/preview-renderer.tsx

```diff
@@ -147,9 +154,11 @@ export function render(init: RenderInit): void {
 
 			return (
 				<ErrorBoundary name={props.name}>
-					<Component {...slots} {...props.properties} data-sketch-name={props.name}>
-						{children.map(child => <PreviewComponent key={child.uuid} {...child} />)}
-					</Component>
+					<Component
+						{...props.properties}
+						{...renderedContents}
+						data-sketch-name={props.name}
```

Comment:
Please leave the distinct `children` / `slots.default` handling in place to signal the slot lands in `children`